Manito/Manito/Domain/Usecase/Common/TextFieldUsecase.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  textFieldUsecase.swift
+//  Manito
+//
+//  Created by 이성호 on 11/3/23.
+//
+
+import Foundation
+
+protocol TextFieldUsecase {
+    func cutTextByMaxCount(text: String, maxCount: Int) -> String
+}
+
+final class TextFieldUsecaseImpl: TextFieldUsecase {
+    func cutTextByMaxCount(text: String, maxCount: Int) -> String {
+        let isOverMaxCount = self.isOverMaxCount(titleCount: text.count, maxCount: maxCount)
+        
+        if isOverMaxCount {
+            let endIndex = text.index(text.startIndex, offsetBy: maxCount)
+            let fixedText = text[text.startIndex..<endIndex]
+            return String(fixedText)
+        }
+        return text
+    }
+    
+    private func isOverMaxCount(titleCount: Int, maxCount: Int) -> Bool {
+        return titleCount > maxCount ? true : false
```

Comment:
어푸르브하고 나서 보이는거지만...
```suggestion
        return titleCount > maxCount
```

이렇게 수정 해도 됩니다!

Reply:
맞습니다!! 저번에 찝어주신 부분인데 여기서는 Usecase로 빼면서 적용이 안됐네여! 반영했습니다.!!

[147dc1c](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/593/commits/147dc1c588b7d108fd3d20ef57e01a931b6c04fd)